README.md

```diff
@@ -82,17 +82,19 @@ score. In the demo we have a preprocessed graph and a test dataset with Uber rid
 
 Launch Docker in the background and from inside the root directory run:
 
+Linux/Mac:
 ```zsh 
 cd kuwala/scripts && sh initialize_core_components.sh && sh run_cli.sh
 ```
-
-***Note***: Some people on Windows experience installation issues with the CLI. If that is the case for you please open 
-an issue. In the meantime you can run the pipelines individually.
+and for Windows:
```

Comment:
Can you add here that you need to use PowerShell on Windows? That's necessary, right?

Reply:
Ahh yes! That would be great if we put some note about PowerShell